token-group/interface/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "spl-token-group-interface"
+version = "0.1.0"
+description = "Solana Program Library Token Group Interface"
+authors = ["Solana Labs Maintainers <maintainers@solanalabs.com>"]
+repository = "https://github.com/solana-labs/solana-program-library"
+license = "Apache-2.0"
+edition = "2021"
+
+[dependencies]
+bytemuck = "1.14.0"
+solana-program = "1.16.3"
```

Just need to get this correct so that the automatic updater works later
```suggestion
solana-program = "1.16.16"
```